README.md

```diff
@@ -368,29 +372,35 @@ These instructions use the [gcloud][gcloud] command-line tool.
 
 1.  Create a Workload Identity **Provider** in that pool:
 
+    **🛑 CAUTION!** Always add an Attribute Condition to restrict entry into the
+    Workload Identity Pool. You can further restrict access in IAM Bindings, but
+    always add a basic condition that restricts admission into the pool. A good
+    default option is to restrict admission based on your GitHub organization as
+    demonstrated below. Please see the [security
+    considerations][security-considerations] for more details.
+
     ```sh
+    # TODO: replace ${PROJECT_ID} and ${GITHUB_ORG} with your values below.
+
     gcloud iam workload-identity-pools providers create-oidc "my-repo" \
       --project="${PROJECT_ID}" \
       --location="global" \
       --workload-identity-pool="github" \
       --display-name="My GitHub repo Provider" \
-      --attribute-mapping="google.subject=assertion.sub,attribute.actor=assertion.actor,attribute.repository=assertion.repository" \
+      --attribute-mapping="google.subject=assertion.sub,attribute.actor=assertion.actor,attribute.repository=assertion.repository,attribute.repository_owner=assertion.repository_owner" \
```

Comment:
nit: should we consider the default to use the owner AND repo? This feels like a more appropriate default.

Reply:
I think a lot of customers have one WIF pool for all things GitHub, so this seemed like a saner default.